data/community_measures/README_borg_dataSources.txt

```diff
@@ -0,0 +1,9 @@
+The borgLayers are dynamically generated so we can change: 
+category names, 
+layer names, 
+enable or disable layers, 
+layer colors, 
+layer categories, 
```

Comment:
Note  sure whether this file should be placed here .. but just leaving a note here as a reminder for us to discuss project structure and files organisation which in general requires a review IMHO. :)